README.md

```diff
@@ -156,7 +187,7 @@ For more information on configuring workload identity on GKE, look at the [offic
 
 Install Velero, including all prerequisites, into the cluster and start the deployment. This will create a namespace called `velero`, and place a deployment named `velero` in it.
 
-**If using a Service Account**:
+**If using a Googl Service Account Key**:
```

Comment:
nit: s/Googl/Google/

Reply:
Thanks for the catch @sseago . . . Pushed a fix for the type 🙏